src/components/Player.vue

```diff
@@ -641,13 +666,53 @@ li.move:not(.from) .player .overlay svg.move {
   top: 5px;
   box-shadow: 0 0 5px black;
   text-overflow: ellipsis;
-  overflow: hidden;
   padding: 0 4px;
 
   #townsquare:not(.spectator) &:hover,
   &.active {
     color: red;
   }
+
+  .pronouns {
```

Comment:
A few issues here:
- the `overflow:hidden` hides the arrow of the box, too
- the pronoun tooltip should have the same height as the player name box
- with the current positioning, the pronoun tooltip is covered by the player menu, so I would show it on the other side, opposite of the player menu
- it can't be inside the player name container because that has `overflow:hidden` so you either have to rework the player name box or move this one outside